go.mod

```diff
@@ -32,6 +34,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/klauspost/pgzip v1.2.5
 	github.com/minio/minio-go/v7 v7.0.31
+	github.com/pkg/errors v0.9.1
```

Comment:
Looks like this accidentally added back the unmaintained `pkg/errors` that was replaced in #7440.

This was fixed/removed again in #13673